src/dbnode/persist/fs/msgpack/encoder.go

```diff
@@ -266,8 +266,11 @@ func (enc *Encoder) encodeIndexSummary(summary schema.IndexSummary) {
 
 func (enc *Encoder) encodeLogInfo(info schema.LogInfo) {
 	enc.encodeNumObjectFieldsForFn(logInfoType)
-	enc.encodeVarintFn(info.Start)
-	enc.encodeVarintFn(info.Duration)
+
+	// Deprecated, have to encode anyways for backwards compatibility, but we ignore the values.
```

Comment:
Can you add a `TODO(v1): remove when we can make backwards incompatible changes with an upgrade to v1`